### DISCUSSION TITLE
GKE 1.22

### DISCUSSION THREAD

**Discussion by sreejesh123** [strimzi/strimzi-kafka-operator#5697] [Q&A] [answered]
need to update CRD to Prepare for GKE 1.22 Open source upstream beta API Removal  .  most of the strimzi crd i am using are apiVersion: apiextensions.k8s.io/v1beta1 (whch is not supported and need to be chagned to /v1), when I tried to change getting some YAML validation error.  is there yaml which uses apiVersion: apiextensions.k8s.io/v1 available anywhere which I use to cross check?

**Comment by scholzj** [comment, marked as answer]
Strimzi is using the `apiextensions.k8s.io/v1` API from 0.23. So you probably need to upgrade to 0.23, 0.24 or 0.25. There is a manual step as part of the CRD upgrade, so read the upgrade docs and follow them.

**Reply by sreejesh123** [reply]
thanks.  I am not able to find version strimzi/kafka:0.23.0-kafka-2.8.0  (or latest) on https://hub.docker.com/r/strimzi/kafka/tags , will you be uploading the image on here?

**Reply by scholzj** [reply]
We do not use Docker Hub anymore already for some time because of its limits it imposes on users. We use Quay.io: https://quay.io/organization/strimzi

PS: If you are just looking for supported versions, they are also listed here: https://strimzi.io/downloads/

**Reply by sreejesh123** [reply]
thank you,   I am not able to find this in quay search.     quay.io/strimzi/kafka-bridge:0.19.0  is this not there ?

**Reply by scholzj** [reply]
If it is not there, it was probably released before we moved to Quay?
